stack.yaml

```diff
@@ -1,41 +1,54 @@
 resolver: lts-23.19
```

Comment:
This is a so much better, thanks! And having the build checked in CI is awesome!